src/store/group-management/saga.ts

```diff
@@ -10,13 +12,16 @@ export function* saga() {
   yield fork(authWatcher);
 
   while (true) {
-    const { startEvent } = yield race({
+    const { startEvent, leaveGroupEvent } = yield race({
```

Comment:
I don't think the leave group needs to be part of the panel management that the add member is part of since it doesn't require rendering anything in the side panel. Can we just have a `takeLatest` above this? If not, then maybe we should pull this panel management bit out to its own thing so we can fork it.

Reply:
makes sense, updated to use `takeLatest`